model/events/LtiAgsListener.php

```diff
@@ -89,9 +89,9 @@ public function onDeliveryExecutionResultsRecalculated(DeliveryExecutionResultsR
         }
     }
 
-    public function onDeliveryExecutionFinish(TestVariablesRecorded $event): void
+    public function onDeliveryExecutionFinish(DeliveryExecutionFinish $event): void
```

Comment:
Do we have some tests for this class / method? If not is it possible to have it?

Reply:
Next time :)